dashboard/app/controllers/hoc_download_controller.rb

```diff
@@ -1,6 +1,10 @@
 class HocDownloadController < ApplicationController
   def index
     @product_name = params[:product]
+    @hoc_url = CDO.code_org_url("/#{@product_name}")
+    @download_urls = {}
+
+    languages = ['Albanian', 'Arabic', 'Azerbaijani', 'Basque', 'Bosnian', 'Bulgarian', 'Catalan', 'Chinese-Taiwan', 'Chinese', 'Croatian', 'Czech', 'Danish', 'Dutch', 'English', 'Finnish', 'French', 'German', 'Greek', 'Hebrew', 'Hungarian', 'Icelandic', 'Indonesian', 'Irish', 'Italian', 'Japanese', 'Korean', 'Latvian', 'Lithuanian', 'Norwegian', 'Norwegian-Nynorsk', 'Polish', 'Portuguese-Brazil', 'Portuguese', 'Romanian', 'Russian', 'Serbian', 'Slovenian', 'Spanish', 'Swedish', 'Turkish', 'Ukrainian', 'Vietnamese']
```

Comment:
Technically this list is in cdo-languages but that's in pegasus DB and this page is on dashboard. Since this page is static, this list isn't changing, and we will not create new offline versions I figured it'd be fine to include here.